src/AWSLambda/Events/APIGateway.hs

```diff
@@ -94,6 +94,19 @@ instance FromJSON RequestIdentity where
       o .:? "user"
 $(makeLenses ''RequestIdentity)
 
+data Authorizer = Authorizer
+  { _aPrincipalId :: !Text
+  , _aClaims :: !(HashMap Text Text)
```

Comment:
I don't know much about Cognito, but are the claims values always guaranteed to be strings or could they be other JSON values? If so, this should be an `Object` rather than `HashMap Text Text`.